src/sonic-device-data/tests/platformJson_checker

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
```

Comment:
Suggest renaming this file to `platform_json_checker`

Reply:
#done